### DISCUSSION TITLE
Cash for new features?

### DISCUSSION THREAD

**Discussion by bsaunder2002** [KillerInk/FreeDcam#222] [General]
Sad to see 2x years since last updates.  Of all the camera apps on f-droid, I think yours has the best features/functionality.

So I bought a Pictar but think I can do better.  What is it?
- a sleeve that makes a cell phone more camera like by 
- buttons that can trigger camera app functions (oddly/coolly driven by ultrasonics)
- a viewfinder hood so you're not staring at a screen glare trying to frame your photo
- a camera app that responds to this.

It's pretty good but...you have to use their app.

So thinking I could to better I:
- bought a $10 viewfinder from wallym...
- bought an Apexel phone lense holder so the Apexel lenses line up perfectly (lenses may be junk we'll see)
-  think I could buy a BT button setup like for steering wheels that has back, forward, up/down/left/right.

Feature one:
Now that viewfinder will  only cover 3ish inches of my screen.  That other company actually shrinks part of the camera view/controls and lets you move it around till under the viewfinder.  I'd like FreeDcam to shrink the whole application and using one of the themes (forget name but not the default one), put the entire app in the viewfinder.  I dont want to pull my head off the viewfinder to adjust exposure, focus, iso, camera im using, ...

Feature two:
bluetooth button integration with the app.  I pair the whatever thing, in your app you see what I'm paired to and I select the button thing.  Then you ask "press the back button", "press the forward button", "press the left", ... and I train FreeDcam to know my controls.

Minor feature:
renaming of cameras so I know what they are...ie rename Camera 1 to "Wide angle".  Hate remembering things.  :) 

So all of this is software.  You dont need to buy anything to write the code for shrink screen and I saw tons of south american river buttons for < $20.

I may not be willing to pay enough...but I am willing to pay...Venmo...risking even prepayment if desired.

Let me know.  Oh yea...prolly miss my vacation next week but...a little extra to dust off code to just have shrink mode?

**Comment by KillerInk** [comment]
Hi
 About feature one
Not sure if that's an easy task.
Freedcam is based on fullscreen and the center. 
I have no idea how the ui acts when I reduce width in mainfragment and shift it to the right. I expect that it get messed up.

About feature two
I don't see a problem to add it. Some basic code is already included to trigger shutter from a wired headset and change manuals.

The thing is, myself currently don't care about that project and I'm busy doing other things. Also that you are willing to donate don't help much to bring me back soon to work on it. 
I put your request on my todo list. 
But if it get added in the next two years, I can't promise. 
Greets

**Reply by bsaunder2002** [reply]
Appreciate the reply.  FreeDCam is the best I've found but it's a lot of work to support all the phones, features, ...
I understand the no time bits.
I'll use what I have and reconsider this after I get back from vacation.  Spent yesterday messing with Apexel cell to lens adapters.  One day I'll learn not to procrastinate.  :)
